packages/web-pkg/src/composables/piniaStores/config/config.ts

```diff
@@ -43,6 +43,9 @@ const defaultOptions = {
   sidebar: {
     shares: {
       showAllOnLoad: false
+    },
+    exif: {
+      showLocation: true
```

Comment:
hum - maybe now I understand why you were reluctant to add the config setting. I had thought of a setting to a specific app. Now it's weird that the setting is in web-pkg and the panel in web-app-files. But having settings for files app would be 💩  as well... yeah well 🙈

Reply:
Yeah, now I built it so let's keep it. ;-)